ui/app/utils/chart-helpers.js

```diff
@@ -20,6 +20,8 @@ export const BAR_WIDTH = 7; // data bar width is 7 pixels
 // Reference for tickFormat https://www.youtube.com/watch?v=c3MCROTNN8g
 export function formatNumbers(number) {
   if (number < 1000) return number;
+  if (number < 1100) return format('.1s')(number);
```

Comment:
Added this so that situations where the yDomain is 0 - ~1500 the ticks will not just show `1k` for all the ticks over 1000. See description gif for example of that happening